src/gql/provider.tsx

```diff
@@ -7,6 +7,8 @@ import { useAuth0 } from "@auth0/auth0-react";
 
 import { store, localSlice } from "@reearth/state";
 import fragmentMatcher from "./fragmentMatcher.json";
+import { SentryLink } from "apollo-link-sentry";
+import { reportError } from "@reearth/sentry";
```

Comment:
Please note the order of import:

1. External modules
2. `@reearth/...`
3. Relative module (children and siblings)